tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Tools/WireMock/PublicBrowse/mappings/default-mapping.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "Guid": "db9e853a-73f0-4605-ad83-240c8e634460",
+    "Priority": 10000,
+    "Request": {
+      "Path": {
+        "Matchers": [
+          {
+            "Name": "WildcardMatcher",
+            "Pattern": "/publicbrowse",
```

Comment:
do you need to mock /back-from-admin too?

Reply:
No because it is a local redirect it will route to `http://host.docker.internal:8070/back-from-admin` as part of the integration tests. So the test will only ensure the route ends with the appropriate value e.g. `/back-from-admin`